pkg/util/glagolitsa/glagolitsa.go

```diff
@@ -54,3 +54,13 @@ const Size = 41
 func LetterByIndex(ind int) string {
 	return script[ind]
 }
+
+// Glagolitsa implement [github.com/nspcc-dev/neofs-contract/deploy.Glagolitsa].
```

Comment:
```suggestion
// Glagolitsa implements [github.com/nspcc-dev/neofs-contract/deploy.Glagolitsa].
```
btw interface name degenerates the abstraction, there is only one glagolitsa. It'd be more clear with `Glagolitsa implements Alphabet`. We cant change it here unfortunately